zipkin-anormdb/src/main/scala/com/twitter/zipkin/storage/anormdb/AnormSpanStore.scala

```diff
@@ -321,21 +350,31 @@ class AnormSpanStore(val db: DB,
     }
   }
 
-  override def getSpanNames(_service: String): Future[Seq[String]] = db.inNewThreadWithRecoverableRetry {
-    val service = _service.toLowerCase // service names are always lowercase!
+  override def getSpanNames(_service: Option[String]): Future[Seq[String]] = db.inNewThreadWithRecoverableRetry {
```

Comment:
I don't have a personal use case for span names across services, so I don't mind backing out of this change